src/main.ts

```diff
@@ -128,7 +129,11 @@ async function run(): Promise<void> {
     core.addPath(toolFolderPath)
     core.debug(`Tool path added to PATH: ${toolFolderPath}`)
   } catch (error) {
-    core.setFailed(error.message)
+    let errorMessage = 'Failed to do something exceptional'
```

Comment:
Why this change?

Reply:
Sorry, testing artifact leftover.

Reply:
No, actually... this is just from downloading, using npm to clean and reinstall with the latest dependencies. I changed the transpiler to vercel from zeit but left everything else the same. I wish I could say more. Feel free to reject the udpate until I can recreate.